### PR TITLE
CORE-5944: Apply CPKDependencies feedback

### PR DESCRIPTION
**Pull request by josephzunigadaly**
- Add .json file extension and an outer object with a format version

### CODE REVIEW

**Review by mbrkic-r3 — APPROVED** [review]
LGTM

File: libs/packaging/packaging-verify/src/main/resources/corda-cpk-2.0.json [hunk 1]
```diff
@@ -3,57 +3,71 @@
...
+            "description": "File hash of third party CPK",
+            "type": "object",
+            "properties": {
+              "algorithm": {
+                "description": "Hash algorithm name",
+                "type": "string"
+              },
+              "fileHash": {
+                "description": "Base64 encoded hash",
+                "type": "string"
```
Comment (kyriathar) [inline libs/packaging/packaging-verify/src/main/resources/corda-cpk-2.0.json:34]: Not for this PR but couldn't we serialize the hash in _algorithm:\<hexString>_ string format and then merge the two fields in one?

**Review by kyriathar — APPROVED** [review]

File: libs/packaging/packaging-verify/src/main/resources/corda-cpk-2.0.json (hunk 1 above)
Comment (josephzunigadaly) [reply libs/packaging/packaging-verify/src/main/resources/corda-cpk-2.0.json:34]: Why would we do that?

Comment (kyriathar) [reply libs/packaging/packaging-verify/src/main/resources/corda-cpk-2.0.json:34]: Also the above format is how we store them everywhere else I think so for consistency also.

Comment (josephzunigadaly) [reply libs/packaging/packaging-verify/src/main/resources/corda-cpk-2.0.json:34]: It would be nice to remove this inner object in the json.

**Pull request merged**